src/core/api.js

```diff
@@ -118,21 +125,25 @@ export function runAPI(express, app, gitCommit, gitBranch, __dirname) {
         try {
             switch (req.params.type) {
                 case 'stream':
-                    if (req.query.t && req.query.h && req.query.e && req.query.t.toString().length === 21
-                    && req.query.h.toString().length === 64 && req.query.e.toString().length === 13) {
-                        let streamInfo = verifyStream(req.query.t, req.query.h, req.query.e);
+                    const q = req.query;
+                    const checkQueries = q.t && q.e && q.h && q.s && q.i;
+                    const checkBaseLength = q.t.length === 21 && q.e.length === 13;
+                    const checkSafeLength = q.h.length === 44 && q.s.length === 344 && q.i.length === 24;
```

Comment:
is this guaranteed?

Reply:
![idk](https://cdn.discordapp.com/emojis/1068869172760682556.webp?size=240&quality=lossless)